Algorithm.Python/BasicTemplateCryptoAlgorithm.py

```diff
@@ -56,4 +56,5 @@ def OnData(self, data):
                 self.Liquidate()
 
             btcHoldings = self.Portfolio.CashBook["BTC"].Amount
-            self.Log("{0} - BTC holdings: {1}".format(str(self.Time), str(btcHoldings)))
\ No newline at end of file
+            usdCash = self.Portfolio.CashBook["USD"].Amount
+            self.Log("{0} - BTC holdings: {1} - USD cash: {2}".format(str(self.Time), str(btcHoldings), str(usdCash)))
```

Comment:
When we use `str.format`, we don't need to convert the objects to string and we can also skip the numbering (unless it is the objects in format are not in the desired ordered):
```python
self.Log("{} - BTC holdings: {} - USD cash: {}".format(self.Time, btcHoldings, usdCash)) 
```